app/models/user.rb

```diff
@@ -0,0 +1,32 @@
+class User < ApplicationRecord
```

Comment:
on password change, refresh `api_token` on user and set current_user by searching `api_token` and logout if user not found.